proto/list_hosts.proto

```diff
@@ -34,10 +34,23 @@ message HostList {
         int32 vlan = 4;
 
         // ACLs
-        repeated string acls = 5;
+        map<string, ACLData> acls = 5;
```

Comment:
as mentioned in the doc, I don't think this can be a map since you lose ordering semantics.

Reply:
it is a list now